tests/unit/test_dist_git_init.py

```diff
@@ -225,6 +225,11 @@
                             },
                         },
                     },
+                    {
+                        "job": "bodhi_update",
+                        "trigger": "commit",
+                        "dist_git_branches": ["fedora-rawhide"],
```

Comment:
Bodhi update for rawhide?

Reply:
shouldn't this be `fedora-branched`?

Reply:
Previously, we generated all the jobs just for rawhide (even if Bodhi was in the end not really needed), but you are right this isn't very fitting now as the `dist-git init` default was changed to support `fast_forward_merge_into` syntax, resulting in doing `pull_from_upstream` for all fedoras. 

However, the updates policy was recently mentioned again in the change proposal discussion, so I am not sure whether we want to change this for both Koji builds and Bodhi updates to be consistent with pull or switch back to rawhide only. WDYT?

Reply:
IIRC Bodhi update for rawhide is done automatically for Bodhi (unless sidetag), won't this cause issues?

Reply:
yes they are, but iirc we handle the exc and just mark those as skipped

Reply:
discussed on standup, agreed on going back to rawhide only also for `pull_from_upstream`